terraform/istio/install_istio.sh

```diff
@@ -44,17 +48,16 @@ kubectl create secret generic cacerts -n istio-system \
     --from-file=samples/certs/cert-chain.pem
 
 # cd back into istio/
-cd ../
+cd ${WORD_DIR}
 
 kubectl label namespace default istio-injection=enabled
 kubectl create clusterrolebinding cluster-admin-binding \
     --clusterrole=cluster-admin \
     --user=$(gcloud config get-value core/account)
 
 # install using operator config - https://istio.io/docs/setup/install/istioctl/#customizing-the-configuration
-INSTALL_PROFILE="./istio_operator.yaml"
+INSTALL_PROFILE="${WORK_DIR}/istio_operator.yaml"
 ${WORK_DIR}/istioctl manifest apply -f ${INSTALL_PROFILE}
```

Comment:
nit, but you probably don't need INSTALL_PROFILE as a variable. You could just do:

`${WORK_DIR}/istioctl manifest apply -f ${WORK_DIR}/istio_operator.yaml`

Reply:
Good idea. Changed :)